src/include/schema.h

```diff
@@ -98,10 +98,9 @@ struct __wt_table {
  * WT_WITH_LOCK --
  *	Acquire a lock, perform an operation, drop the lock.
  */
-#define	WT_WITH_LOCK(session, ret, lock, flag, op) do {			\
+#define	WT_WITH_LOCK(session, ret, nowait, lock, flag, op) do {		\
```

Comment:
> How about we just make this WT_WITH_LOCK_NOWAIT and avoid the extra argument and the else?

I didn't do that because it implies the waiting version of the lock macros no longer need a `ret` variable and so they all change, and I figured the compiler would drop the unused code since it's testing a constant.

I'll go ahead and push that change, if you don't like it just revert it.